From 34d4304e7bc264b2db99bfdb076f243458936d93 Mon Sep 17 00:00:00 2001
From: Stella Laurenzo <stellaraccident@gmail.com>
Date: Fri, 11 Apr 2025 18:15:02 -0700
Subject: [PATCH] Explicitly include GoogleTest helpers.

* These automatically get included in some legacy find modules but are not universally available without an include.
---
 test/gtest/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/test/gtest/CMakeLists.txt b/test/gtest/CMakeLists.txt
index 7d7fbec71..594ba3fda 100644
--- a/test/gtest/CMakeLists.txt
+++ b/test/gtest/CMakeLists.txt
@@ -1,6 +1,8 @@
 find_package(GTest REQUIRED)
 find_package(rocblas)
 
+include(GoogleTest)
+
 if(MIOPEN_USE_HIPBLASLT)
   find_package(hipblas REQUIRED PATHS /opt/rocm $ENV{HIP_PATH})
   find_package(hipblaslt REQUIRED PATHS /opt/rocm $ENV{HIP_PATH})
-- 
2.43.0

